canisters/src/vts/src/lib.rs

```diff
@@ -186,9 +190,12 @@ struct Vehicle {
 }
 impl_storable!(Vehicle);
 
-#[derive(CandidType, Deserialize)]
+#[derive(CandidType, Deserialize, Clone)]
```

Comment:
Clone can be deleted I think as you do not need invoice.clone() below.

Reply:
I still use clone in the method `get_invoice`